resources/quotes.json

```diff
@@ -11,6 +11,10 @@
     "quote": "All warfare is based on deception.",
     "tooltip": "Sun Tzu said that!"
   },
+  {
+    "quote": "An eye for an eye makes the whole world blind",
```

Comment:
Most quotes have punctuation at the end, do we really want to make an exception here?